### ISSUE TITLE
Log scales doesn't work with bar charts

### ISSUE THREAD

**Issue by operman18** [metabase/metabase#10384]
Hi,

I am using metabase on elasticbeanstalk. And for the past few months, I have been struggling with this issue. I don't know what is the expected behavior here, but I am seeing an empty plot when I'm trying to use the logarithmic scale with bar charts. I don't have missing, null or negative values (or any value lower than one).

I think metabase should be able to
    + handle missing, null, negative or at least have an improved logging.
    + plot all values, the same way that it's plotting for the power scale.

P.S. Everything seems to be working fine when I use a regular line

![log_scales_area](https://user-images.githubusercontent.com/30746044/61575144-5d8f7100-aaf2-11e9-9599-f1a9629be098.png)
![log_scales](https://user-images.githubusercontent.com/30746044/61575145-5e280780-aaf2-11e9-959e-9dec987f599e.png)
![power_scales](https://user-images.githubusercontent.com/30746044/61575146-5ec09e00-aaf2-11e9-98ac-1b3b1bacb6c2.png)

**Comment by paulrosenzweig** [comment]
@operman18 I'm unable to reproduce exactly the issues in your screenshots, but I did find a bug in stacked bar charts with log scales. I created a branch with a fix for that: `fix-10384`

Are you able to check if that branch fixes your problem? If you don't have a development environment set up, it'd be helpful to have more information so I can reproduce your issue on my computer. Are you able to reproduce this bug with the sample dataset? Or, can you write a small SQL query that reproduces it? e.g. something like this:

```sql
select 'a' x, 3 y, 2 z
union all
select 'b' x, 3 y, 3 z
union all
select 'c' x, 3 y, 4 z
```

**Comment by operman18** [comment]
@paulrosenzweig   Same, but it looks like when I remove zero from the axis it works.

![output-4](https://user-images.githubusercontent.com/30746044/65154279-6a2e3800-da55-11e9-9a3b-391457d5e591.gif)

**Comment by paulrosenzweig** [comment]
@operman18 ok for the issues shown in that gif, #10911 should fix them. I'll set that PR to close this issue. Thanks!

**Comment by paulrosenzweig** [comment]
Fixed by #10911
